tests/fillers/build_json.py

```diff
@@ -13,7 +13,7 @@
     filler_formatter,
 )
 
-from eth.utils.hexadecimal import (
+from eth_utils import (
```

Comment:
I feel bad about nitpicking this stuff and especially because I missed to comment on in the previous run but can you please make sure that this import is placed after the `eth_hash` import.

The import order rules are described here: https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#overall-ordering

Reply:
That was a good catch anyways, glad I learned about this rules.